.github/workflows/publish-helm-chart.yml

```diff
@@ -75,6 +75,61 @@ jobs:
             run: |
               helm lint ${{matrix.chartpath}}
 
+          - name: Helm Chart Version Check
+            run: |
+              # old_chart_version diffs commit that triggered workflow against previous 
+              # commit. Pulls out version number of the existing chart or "old chart".
+              # The git diff gives spits out both version numbers, head -2 | tail -1
+              # Grabs the value on the second line. 
+              old_chart_version=$(git diff ${{ github.sha }} \
+               ${{github.event.before}} -- ${{matrix.chartpath}} | 
```

Comment:
I'm not sure `github.event.before` is sufficient here, it may only apply to pushes and not PRs (https://stackoverflow.com/questions/61860732/how-can-i-get-the-previous-commit-before-a-push-or-merge-in-github-action-workfl).

It's kinda why we have all this goofy logic: https://github.com/osg-htc/helm-charts/pull/22/files#diff-dce247c65e791dfbb2975e023ffc0d20d556f48be798bc6c462ba808ccbbc26fR28-R36, though the GHA stuff moves fast so there may be better ways to do it now!

What I'd do is export the `BASE` into a `GITHUB_OUTPUT` variable and use that here instead of reproducing the logic